framework/wazuh/core/tests/test_utils.py

```diff
@@ -217,70 +217,83 @@ def test_cut_array_ko(limit, offset, expected_exception):
-                               {'item': 'value_1', 'datetime': '2018-05-15T12:34:12Z'}],
-                              'datetime>2017-10-26;datetime<2018-05-15T12:34:12.001000Z', None, None, None, None, None,
-                              [{'item': 'value_1', 'datetime': '2018-05-15T12:34:12Z'}], 1),
-
-                             # Test cases with filters, limit and search
-                             ([{'item': 'value_1'}, {'item': 'value_2'}, {'item': 'value_3'}],
-                              None, {'item': 'value_1'}, 1, None, None, None,
-                              [{'item': 'value_1'}], 1),
-
-                             ([{'item': 'value_1'}, {'item': 'value_1'}, {'item': 'value_3'}],
-                              None, None, 1, 'e_1', None, None,
-                              [{'item': 'value_1'}], 2),
-
-                             ([{'item': 'value_1'}, {'item': 'value_1'}, {'item': 'value_3'}],
-                              None, None, 2, 'e_1', None, None,
-                              [{'item': 'value_1'}, {'item': 'value_1'}], 2),
-
-                             # Test cases with sort
-                             ([{'item': 'value_2'}, {'item': 'value_1'}, {'item': 'value_3'}],
-                              None, None, None, None, None, ['item'],
-                              [{'item': 'value_1'}, {'item': 'value_2'}, {'item': 'value_3'}], 3),
-
-                             # Complex test cases
-                             ([{'item': 'value_1', 'datetime': '2017-10-25T14:48:53.732000Z', 'component': 'framework'},
-                               {'item': 'value_1', 'datetime': '2018-05-15T12:34:12.544000Z', 'component': 'API'}],
-                              'datetime~2017', {'item': 'value_1'}, 1, 'frame', None, None,
-                              [{'item': 'value_1', 'datetime': '2017-10-25T14:48:53.732000Z',
-                                'component': 'framework'}], 1),
-
-                             ([{'item': 'value_1', 'datetime': '2017-10-25T14:48:53.732000Z', 'component': 'framework'},
-                               {'item': 'value_1', 'datetime': '2018-05-15T12:34:12.544000Z', 'component': 'API'}],
-                              'datetime~2019', {'item': 'value_1'}, 1, None, None, None,
-                              [], 0),
-
-                             ([{'item': 'value_1', 'datetime': '2017-10-25T14:48:53.732000Z', 'component': 'framework'},
-                               {'item': 'value_1', 'datetime': '2018-05-15T12:34:12.544000Z', 'component': 'API'}],
-                              'datetime~2017', {'item': 'value_1'}, 1, None, ['component', 'item'], None,
-                              [{'item': 'value_1', 'component': 'framework'}], 1),
-                         ])
-def test_process_array(array, q, filters, limit, search_text, sort_by, select, expected_items, expected_total_items):
+@pytest.mark.parametrize(
```

Comment:
Since `distinct` is of type `bool`, consider defining `False` instead of `None` for false cases, or are there any cases where `process_array` is called with `None`?

Reply:
> are there any cases where process_array is called with None?

No, all `distinct` default values were added as `False` and I changed the existing ones from `None` to `False` as well. I will change the ones from the test as suggested.